rewrite-maven/src/main/java/org/openrewrite/maven/internal/InsertDependencyComparator.java

```diff
@@ -64,7 +64,8 @@ public InsertDependencyComparator(List<? extends Content> existingDependencies,
 
     @Override
     public int compare(Content o1, Content o2) {
-        return positions.get(o1).compareTo(positions.get(o2));
+        Float anotherFloat = positions.get(o2);
+        return anotherFloat == null ? 0 : positions.get(o1).compareTo(anotherFloat);
```

Comment:
Originally had `1` here, but then that changed the order for the comments below; figured `0` is neutral. :/